lucien-ios/Subclasses/BottomBorderTextField.swift

```diff
@@ -35,7 +35,7 @@ final class BottomBorderTextField: UITextField {
     }
 
     private func createBottomBorder() {
-        bottomBorder.frame = CGRect(x: 0.0, y: frame.height - 3.0, width: frame.width - LucienConstants.textFieldBorderOffset, height: 1.0)
+        bottomBorder.frame = CGRect(x: 0.0, y: frame.height - 3.0, width: frame.width, height: 1.0)
```

Comment:
The 3.0 should be saved in a private struct called `Constants` in this file, that way it's obvious what it is, and it's easy to update it if needed.